src/provider/controller/NotificationCenter.ts

```diff
@@ -72,6 +72,13 @@ export class NotificationCenter extends AsyncInit {
         this._trayIcon.onLeftClick.add(() => {
             new ToggleCenterVisibility(ToggleCenterVisibilitySource.TRAY).dispatch(this._store);
         });
+        try {
```

Comment:
How come this is in a try/catch and not just a promise catch on the end of `fin.GlobalHotkey.register`?

Reply:
I don't think we've ever used promise catch anywhere else in the project. try-catch seems to be the convention (?)
I can give line 64 of this file or persistor.ts for a few examples.